users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -371,18 +656,24 @@ def process_args(args: Dict[str, Any]):
                     "conv_pad_seq_len_to_power": 1.5,
                     "watch_memory": True,
                     "accum_grad_multiple_step": 2,
+                    "preload_from_files": {
+                        "existing-model": {
+                            "filename": nn_system.train_jobs["conformer_bs2x5k_scf_baseline"].out_checkpoints[24],
+                            "init_for_train": True,
+                        }
+                    },
                 },
                 **returnn_args,
             },
-            feature_args=feature_args,
+            feature_args={"class": "ScfNetwork", "size_tf": 256 // 2, "stride_tf": 10 // 2},
```

Comment:
Why is `wave_norm` not used here?

Reply:
I had the idea that wave_norm could be one of the reasons why preemphasis perturbation did not perform well. I want to try running it without.

Reply:
But wouldn't you want to check this by running both corresponding experiments?

Reply:
I will add it, but than I also have to add another baseline training with just wave_norm as a checkpoint